recipes/segway/meta.yaml

```diff
@@ -1,45 +1,34 @@
+{% set name = "segway" %}
+{% set version = "2.0.3" %}
+
 package:
-  name: segway
-  version: "2.0.2"
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-  url: https://pypi.python.org/packages/16/a7/e1660022578ea53c2c1ee5958e6a7ef9868258fb8ac36e52bc2cda344e30/segway-2.0.2.tar.gz
-  md5: bb7bca1faf7a97dc0208fe944b577d66
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b2f2c97e0ff9d064b8271554e60e954764354d04237876766e55e4e484cbb547
```

Comment:
If you have `{{ version }}` then you should also have `{{ sha256 }}` or so set, so that they are updated together.